apps/api/src/app/rate-limiting/guards/throttler.guard.ts

```diff
@@ -174,15 +177,7 @@ export class ApiRateLimitInterceptor extends ThrottlerGuard implements NestInter
         apiServiceLevel
       )
     );
-    res.rateLimitPolicy = {
-      limit,
-      windowDuration,
-      burstLimit,
-      algorithm,
-      apiRateLimitCategory,
-      apiRateLimitCost,
-      apiServiceLevel,
-    };
+    res.rateLimitPolicy = rateLimitPolicy;
```

Comment:
This line isn't required anymore, it was only used for the `customProps` downstream logging context. We should remove it.
```suggestion
```